Taskfile.yml

```diff
@@ -206,59 +212,96 @@ tasks:
     generates: ["{{.CHECKSUM_FILE}}"]
 
   webui-nodejs:
```

Comment:
Not a big deal to leave it since we'll soon clean up the ordering of tasks, but we could move `webui-nodejs` after `webui-node-modules` to maintain an alphabetical ordering of these artifact-building internal tasks.